docs/section10/10-14.md

```diff
@@ -0,0 +1,177 @@
+                        - Set expected_baseline_eff_b equal to 78% AFUE: `expected_baseline_eff_b = 0.78`
+                        - Check if the modeled efficiency meets the expected baseline efficiency: `efficiency_matches_acceptable_value = expected_baseline_eff_b == modeled_efficiency_b`                  
+                    - Check efficiency_matches_acceptable_value equals True, if true then go to the Rule Assertion, if it equals FALSE perform logic below; `if efficiency_matches_acceptable_value == FALSE:` 
+                        - Reset modeled_efficiency_b to equal Null: `modeled_efficiency_b = Null`  
+                        - Loop through each of the efficiency_metric_types associated with the heating system until THERMAL_EFFICIENCY is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                            - Check if the efficiency metric equals THERMAL_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "THERMAL_EFFICIENCY":` 
+                                - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`
+                            - Else, do nothing and move on to the next metric: `Else:`   
+                        - Check that modeled_efficiency_b does not equal Null: `if modeled_efficiency_b != Null:`    
+                            - Set expected_baseline_eff_b equal to 80% Et: `expected_baseline_eff_b = 0.8`
+                            - Check if the modeled efficiency meets the expected baseline efficiency: `efficiency_matches_acceptable_value = expected_baseline_eff_b == modeled_efficiency_b`                       
+        - Else, system type is 4:`Else:` 
+            - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`   
+            - Get list of zones that the HVAC system serves (this is to get the zone associated with sys 3 or 4): `zone_list_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["ZONE_LIST"]`  
+            Get the cooling capacity (needed for determining heating efficiency for system 4)
+            - Check if rated cooling capacity does not equal null, if it does not equal null then set the cooling capacity equal to this data element's value: `if cooling_system_b.rated_total_cool_capacity != Null:`
+                - Set cooling capacity equal to the rated cooling capacity: `total_cool_capacity_b = cooling_system_b.rated_total_cool_capacity`
+            - Else if, check that design cooling capacity does not equal null:`Elif cooling_system_b.design_total_cool_capacity != Null:`  
+                - Set cooling capacity equal to the design cooling capacity: `total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`
```

Comment:
We allow either design or rated. Are we still okay with this now that we have adopted a stricter approach generally speaking?

Reply:
I think it is good that the logic prioritizes rated capacity, but uses design capacity if the rated capacity does not exist.